app/assets/javascripts/fortitude/components/flashbar.js

```diff
@@ -4,18 +4,14 @@ title: Flashbar
 name: js-flashbar
 category: Javascript
 ---
-Flashbars are used to inform a user of something after an action and can optionally be dismissed
+Flashbars are used to inform a user of something after an action. The javascript component allows you to hide the flashbar when an element is clicked, such as a close button.
 
-<div class="note note--info">
-  <p class="styleguide">Flashbars don't need JavaScript by default, but if you want the flashbar to dismiss without an extra call to the server then you'll want to use the JS Component</p>
-  <p class="styleguide">You must provide the following data attributes if you want the JS component to work</p>
-  <ul class="styleguide">
-    <li><code class="styleguide">data-ft-flashbar</code> on the <code class="styleguide">.flashbar</code> element</li>
-    <li><code class="styleguide">data-ft-flashbar-hide</code> on the <code class="styleguide">.flashbar__hide</code> element</li>
-  </ul>
-</div>
+### Data Attributes
```

Comment:
I think `.note` is great for calling out stuff that's special about a particular component, or gotchas and optional behavior that aren't obvious. Since we have data attributes and events on every js component, I thought it was too noisy to have callouts for every one. Cleaner + easier to read with plain sections.